luigi/batch_notifier.py

```diff
@@ -108,21 +113,21 @@ def _format_tasks(self, tasks):
             return six.u('- {}').format(six.u('\n  ').join(lines))
 
     def _owners(self, owners):
-        return self._default_owner | set(owners or ())
+        return self._default_owner | set(owners)
 
-    def add_failure(self, task_name, family, unbatched_args, expl, owners=None):
+    def add_failure(self, task_name, family, unbatched_args, expl, owners):
```

Comment:
They're not that different, just a `[]` added to every call. `owners=[]` is an anti-pattern because when owners goes to the default, it's always the same list. So if you append to the list after accepting a default argument, the default is no longer empty. So it's a choice between doing `owners=None` and starting the function with

```
if owners is None:
    owners = []
```

and adding an argument in the tests. Since we don't need a default argument, I'd prefer not to do that in the actual code just to make the tests look slightly nicer.